rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/fragment/CovidPatientRegisterFragment.java

```diff
@@ -49,9 +51,11 @@ protected String getDefaultSortQuery() {
     @Override
     public void onSyncComplete(FetchStatus fetchStatus) {
         super.onSyncComplete(fetchStatus);
-        RDTApplication.getInstance()
-                .getContext()
-                .allSharedPreferences()
-                .savePreference(Constants.Preference.CTS_LATEST_SYNC_TIMESTAMP, String.valueOf(System.currentTimeMillis()));
+        AllSharedPreferences allSharedPreferences = RDTApplication.getInstance().getContext().allSharedPreferences();
+        allSharedPreferences.savePreference(Constants.Preference.CTS_LATEST_SYNC_TIMESTAMP, String.valueOf(System.currentTimeMillis()));
+
+        if (getActivity() instanceof PatientRegisterActivity) {
+            ((PatientRegisterActivity) getActivity()).updateSyncDate(allSharedPreferences);
+        }
```

Comment:
We should use a custom broadcast instead and register a broadcast receiver in the PatientRegisterActivity that updates the timestamp.

Reply:
done